src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -4,8 +4,267 @@
+        public System.Numerics.Tensors.ReadOnlyTensorSpan<T> this[scoped System.ReadOnlySpan<System.Buffers.NRange> ranges] { get { throw null; } }
+        public ref readonly T this[scoped System.ReadOnlySpan<nint> indexes] { get { throw null; } }
+        [System.Diagnostics.CodeAnalysis.UnscopedRefAttribute]
+        public System.ReadOnlySpan<nint> Lengths { get { throw null; } }
+        public int Rank { get { throw null; } }
+        [System.Diagnostics.CodeAnalysis.UnscopedRefAttribute]
+        public System.ReadOnlySpan<nint> Strides { get { throw null; } }
+        public static System.Numerics.Tensors.ReadOnlyTensorSpan<T> CastUp<TDerived>(System.Numerics.Tensors.ReadOnlyTensorSpan<TDerived> items) where TDerived : class?, T? { throw null; }
+        public void CopyTo(System.Numerics.Tensors.TensorSpan<T> destination) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("Equals() on ReadOnlyTensorSpan will always throw an exception. Use the equality operator instead.")]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
+        public override bool Equals(object? obj) { throw null; }
+#pragma warning restore CS0809 // Obsolete member overrides non-obsolete member
+        public System.Numerics.Tensors.ReadOnlyTensorSpan<T>.Enumerator GetEnumerator() { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("GetHashCode() on ReadOnlyTensorSpan will always throw an exception.")]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
+        public override int GetHashCode() { throw null; }
+#pragma warning restore CS0809 // Obsolete member overrides non-obsolete member
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public ref readonly T GetPinnableReference() { throw null; }
+        public static bool operator ==(System.Numerics.Tensors.ReadOnlyTensorSpan<T> left, System.Numerics.Tensors.ReadOnlyTensorSpan<T> right) { throw null; }
+        public static implicit operator System.Numerics.Tensors.ReadOnlyTensorSpan<T> (T[]? array) { throw null; }
+        public static bool operator !=(System.Numerics.Tensors.ReadOnlyTensorSpan<T> left, System.Numerics.Tensors.ReadOnlyTensorSpan<T> right) { throw null; }
+        public System.Numerics.Tensors.ReadOnlyTensorSpan<T> Slice(scoped System.ReadOnlySpan<System.Buffers.NIndex> indexes) { throw null; }
+        public System.Numerics.Tensors.ReadOnlyTensorSpan<T> Slice(scoped System.ReadOnlySpan<System.Buffers.NRange> ranges) { throw null; }
+        public override string ToString() { throw null; }
+        public bool TryCopyTo(System.Numerics.Tensors.TensorSpan<T> destination) { throw null; }
+        public bool TryFlattenTo(System.Span<T> destination) { throw null; }
```

Comment:
We have TryFlattenTo, but look to be missing FlattenTo

Reply:
Its in the code itself so let me make sure its getting added to the ref correctly.